source/_components/obihai.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "Obihai"
+description: "Instructions on how to integrate your Obihai device into Home Assistant."
+logo: obitalk.png
+ha_iot_class: "Local Polling"
+ha_category:
+  - Sensor
+ha_release: 0.99
+redirect_from:
+  - /components/sensor.obihai/
+---
+
+The `obihai` integration allows you to view the call status for your [Obihai devices](https://www.obitalk.com/info/products#home_section).
+
+To enable `obihai` in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: obihai

```

Comment:
Only required items in the first example

Reply:
done